apps/rehabstod/src/components/Form/hooks/useInputStyle.ts

```diff
@@ -13,9 +13,13 @@ const getStyle = ({ error, disabled, bright }: Record<string, boolean | undefine
     return 'bg-white border-accent-40'
   }
 
+  if (flex) {
+    return 'bg-secondary-95 border-accent-40 flex'
+  }
+
   return 'bg-secondary-95 border-accent-40'
 }
 
 export function useInputStyle(props: Record<string, boolean | undefined>): string {
-  return classNames('text-neutral-20 my-3 box-border flex w-full rounded border text-left overflow-hidden text-ellipsis', getStyle(props))
+  return classNames('text-neutral-20 my-3 box-border w-full rounded border text-left truncate', getStyle(props))
```

Comment:
Should truncate be conditional depending on the prop?

Reply:
I thought it made sense to always add truncate, and instead make the flex conditional since its used by the datePicker.